MQuince/MQuince.Services.Tests/AppointmentTests.cs

```diff
@@ -316,7 +318,62 @@ public void Get_free_appointments_return_appointments()
             Assert.True(CompareAppointmentsDTO(output, appointments));
         }
 
-		private bool CompareAppointmentsDTO(IEnumerable<Appointment> output, IEnumerable<AppointmentDTO> appointments)
+        [Fact]
+        public void Cancel_appointment()
+        {
+            AppointmentService _appointmentService = new AppointmentService(CreateStubRepository(), workTimeService);
+
+            Guid appointmentId = Guid.Parse("08d89d75-3533-47ac-80a1-6cd77742dd25");
+
+            _appointmentService.CancelAppointment(appointmentId);
+            IdentifiableDTO<AppointmentDTO> canceledAppointment = _appointmentService.GetById(appointmentId);
+
+            Assert.True(canceledAppointment.EntityDTO.IsCanceled);
+        }
```

Comment:
Problem with this test is that is dependant of time when test is started. Eg. You are trying to cancel appointment which time is less than 48 hours from DateTime.Now . This probably worked for you last night beacause it was more than 48 hours from appointment time.
My suggestion for you is to add new appointment in Stub Repository (that way you are not interrupting other tests), setting date time depending on DateTime.Now rather than harcoding values.

Reply:
Also you have other tests to write. Eg. You did not test when appointment is less than 48 hours from DateTime.Now and you try to cancel it, when appointment passed and you try to cancle it, etc..